src/actions/guides/database/models.cr

```diff
@@ -118,6 +118,15 @@ class Guides::Database::Models < GuideAction
     end
     ```
 
+    Note that should you remove the `timestamps` default column and manually add `created_at` and/or `updated_at` to specific models, you will need to leverage the `autogenerated: true` flag in order to preserve the automatic default behavior. An example can be seen in the default `timestamps` macro implementation in Avram:
```

Comment:
This sentence confused me. I may have to think about this a bit 🤔

Reply:
Lol reading this again, I agree. I'll take another swing at the language this week unless y'all can think of a succinct way to put it.

Reply:
*summoning* @edwardloveall 😬

Reply:
Can I talk though this a little? I think what you're trying to say could be phrased like:

> If you remove `timestamps` from these default columns and you still want the automatic default behavior, you'll need to ensure that:
>
> * Add `created_at` and/or `updated_at` columns to specific models
> * Make sure they are annotated with `autogenerated: true`
>
> Here's an example from Avram:

---

I'm not totally sure what that does or if I got the instructions right. Seems like the problem here is mostly that it's a run-on sentence with some conditions at the start _and_ the end. Breaking it up helps.

Reply:
Yeah, I think the whole purpose of the `autogenerated: true` is that it makes it so the SaveOperation doesn't yell when you don't pass those fields. If a column is required, but the user will never pass the value to a SaveOperation, then we add the `autogenerated: true`, and internally it all handles itself. (same with the primary key I think).

Reply:
Ah neat. That all makes sense. Might be worth adding a bit in there about that:

> If these fields are not marked as `autogenerated: true` operations will complain about these required fields being missing. The operation can't know that the database will generate these values automatically, so we `autogenerated: true` to give it that information.